### PR TITLE
Release v0.21.0

### PR DESCRIPTION
**Pull request by DmitriyMolch**
https://cerenetwork.atlassian.net/browse/CBI-2193

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "name": "cerestats",
-  "version": "0.20.0",
+  "version": "0.21.0",
```
Comment (AndreiNavoichyk) [inline package.json:4]: Update all `package.json` files. You can find some in the directories: api, backend, etc.

Comment (DmitriyMolch) [reply package.json:4]: fixed

File: api/src/services/healthService.js [hunk 2]
```diff
@@ -1,140 +1,261 @@
...
+  
+  let healthErrors = [];
+  await Promise.all(
+    networks.map(async network => {      
+      const [best, finalized] = await Promise.all([
+        cereNetworkService.getFinalizedBlock(network),
+        cereNetworkService.getBestBlock(network)
+      ]);      
+      const bestBlockNumberDiff = best - finalized;
+      if (bestBlockNumberDiff > cereConfig.bestBlockNumberDiff) {
```
Comment (AndreiNavoichyk) [inline api/src/services/healthService.js:141]: `cereConfig.bestBlockNumberDiff` should be updated in the k8s configs?

File: api/src/services/healthService.js [hunk 3]
```diff
@@ -1,140 +1,261 @@
...
+  let healthErrors = [];
+  await Promise.all(
+    networks.map(async network => {
+      const { block } = await cereNetworkService.getLatestBlock(network);
+      const [previousTime, lastTime] = await Promise.all([
+        cereNetworkService.getBlockTime(network, block.header.number - 1),
+        cereNetworkService.getBlockTime(network, block.header.number),
+      ]);
+      const timeDiff = lastTime - previousTime;
+      if (timeDiff > cereConfig.blockTimeDiffMs) {
```
Comment (AndreiNavoichyk) [inline api/src/services/healthService.js:176]: `cereConfig.blockTimeDiffMs` should be updated in the k8s config?

File: api/src/services/healthService.js (hunk 2 above)
Comment (DmitriyMolch) [reply api/src/services/healthService.js:141]: yes

File: api/src/services/healthService.js (hunk 3 above)
Comment (DmitriyMolch) [reply api/src/services/healthService.js:176]: yes

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**